### ISSUE TITLE
Dead project?

### ISSUE THREAD

**Issue by vojta-horanek** [hannesschulze/conecto#30]
Is this project dead? There doesn't seem to be any activity for the past few months. If this is dead, is there a different similar project?

**Comment by GabenCrying** [comment]
yes its dead

no theres nothing else :(

**Comment by Hedronmx** [comment]
rip

**Comment by hannesschulze** [comment]
Hi, I am afraid there won't be any updates in the near future. I am currently not using Linux on the desktop anymore which makes it hard to develop this app.

However, I still have hopes to revive this project later this or next year. I have already started implementing an updated rewrite of the communication library and a new GUI in C++ last year (on the `new-ui` and `rewrite` branches). I am probably going to delete this repository for now because:
 * hosting two essentially different applications (although one version uses some parts of the other) is difficult and
 * `master` currently just contains some cleanup from the original version.

For the time being, I am not aware of any alternatives to use on elementary OS.

**Comment by quentincaffeino** [comment]
It might be better to just archive project since code might be useful for somebody later.

Anyways, thanks for the awesome project and good luck :)

**Comment by hannesschulze** [comment]
Yes, that would be another option. I think it would be best to merge the latest WIP changes into master and update the README beforehand so the archived state at least more accurately reflects the last state of the project for now.
